pywinauto/unittests/test_application.py

```diff
@@ -47,6 +47,9 @@
 import mock
 import six
 
+from pywinauto.controls.hwndwrapper import HwndWrapper
+from pywinauto.controls.uiawrapper import UIAWrapper
```

Comment:
If you need these wrappers, maybe move these tests to `test_hwndwrapper.py` and `test_uiawrapper.py`? It shouldn't add new dependent modules.

Reply:
It also breaks some tests because backend specific tests have check if comtypes is installed. Some runs have UIA_SUPPORT=NO (comtypes not installed). It is designed specifically to detect such architecture issues.